### PR TITLE
Fixes BZ1383015 - Revert openshift.node.nodename changes

### PR DESCRIPTION
**Pull request by sdodson**
Fixes Bug 1383015
Reverts #2561 
Reverts #2501 

We've not been able to reproduce this but we know the change that introduced the problems. So we'll revert them on release-1.3 and investigate further on master.

### CODE REVIEW

**Comment by sdodson** [comment]
aos-ci-test

File: playbooks/byo/openshift-cluster/upgrades/docker/docker_upgrade.yml [hunk 1]
```diff
@@ -19,29 +19,29 @@
...
     when: l_docker_upgrade is defined and l_docker_upgrade | bool
 
   - name: Set node schedulability
     command: >
-      {{ openshift.common.client_binary }} adm manage-node {{ openshift.node.nodename }} --schedulable=true
+      {{ openshift.common.admin_binary }} manage-node {{ openshift.common.hostname | lower }} --schedulable=true
     delegate_to: "{{ groups.oo_first_master.0 }}"
     when: openshift.node.schedulable | bool
-    when: l_docker_upgrade is defined and l_docker_upgrade | bool and inventory_hostname in groups.oo_nodes_to_upgrade and openshift.node.schedulable | bool
+    when: l_docker_upgrade is defined and l_docker_upgrade | bool and inventory_hostname in groups.oo_nodes_to_config and openshift.node.schedulable | bool
```
Comment (detiber) [inline playbooks/byo/openshift-cluster/upgrades/docker/docker_upgrade.yml]: The changes here seem unrelated.

File: playbooks/common/openshift-cluster/redeploy-certificates.yml [hunk 2]
```diff
@@ -224,24 +224,24 @@
...
       --evacuate --force
     delegate_to: "{{ groups.oo_first_master.0 }}"
     when: openshift_certificates_redeploy_ca | default(false) | bool and was_schedulable | bool
 
   - name: Set node schedulability
     command: >
-      {{ openshift.common.client_binary }} adm --config={{ hostvars[groups.oo_first_master.0].mktemp.stdout }}/admin.kubeconfig
-      manage-node {{ openshift.node.nodename }} --schedulable=true
+      {{ openshift.common.admin_binary }} --config={{ hostvars[groups.oo_first_master.0].mktemp.stdout }}/admin.kubeconfig
+      manage-node {{ openshift.common.hostname | lower }} --schedulable=true
```
Comment (detiber) [inline playbooks/common/openshift-cluster/redeploy-certificates.yml]: This file looks unrelated to the change as well.

File: playbooks/common/openshift-cluster/upgrades/upgrade_control_plane.yml [hunk 3]
```diff
@@ -1,93 +1,39 @@
 ---
 ###############################################################################
-# Upgrade Masters
+# The restart playbook should be run after this playbook completes.
```
Comment (detiber) [inline playbooks/common/openshift-cluster/upgrades/upgrade_control_plane.yml]: probably want to have @dgoodwin validate this change.

File: roles/openshift_cloud_provider/templates/openstack.conf.j2 [hunk 4]
```diff
@@ -2,11 +2,6 @@
 auth-url = {{ openshift_cloudprovider_openstack_auth_url }}
 username = {{ openshift_cloudprovider_openstack_username }}
 password = {{ openshift_cloudprovider_openstack_password }}
-{% if openshift_cloudprovider_openstack_domain_id is defined %}
-domain-id = {{ openshift_cloudprovider_openstack_domain_id }}
-{% elif openshift_cloudprovider_openstack_domain_name is defined %}
-domain-name = {{ openshift_cloudprovider_openstack_domain_name }}
-{% endif %}
```
Comment (detiber) [inline roles/openshift_cloud_provider/templates/openstack.conf.j2]: we'll probably want to attempt to keep the domain_id and domain_name settings here.

File: playbooks/common/openshift-cluster/redeploy-certificates.yml (hunk 2 above)
Comment (sdodson) [inline playbooks/common/openshift-cluster/redeploy-certificates.yml]: Yeah, fixing some merge conflicts now.

**Comment by sdodson** [comment]
aos-ci-test

**Comment by sdodson** [comment]
@detiber ok, i think i've pared it down to only the required changes while resolving merge/conflicts properly

**Comment by detiber** [comment]
@sdodson we'll probably need to create a bug to track fixing the issue in master.
